ts/src/pro/coinex.ts

```diff
@@ -21,22 +19,24 @@ export default class coinex extends coinexRest {
                 'watchTicker': true,
                 'watchTickers': true,
                 'watchTrades': true,
-                'watchMyTrades': false, // can query but can't subscribe
+                'watchMyTrades': true,
                 'watchOrders': true,
                 'watchOrderBook': true,
-                'watchOHLCV': true, // only for swap markets
-                'fetchOHLCVWs': true,
+                'watchOHLCV': false,
```

Comment:
is this a regression? weird

Reply:
Yeah it looks like it, maybe they'll bring back support later